katharsis-client/src/main/java/io/katharsis/client/KatharsisClient.java

```diff
@@ -461,4 +448,8 @@ public HttpAdapter getHttpAdapter() {
 	public ModuleRegistry getModuleRegistry() {
 		return moduleRegistry;
 	}
+
+	public ClientDocumentMapper getDocumentMapper() {
```

Comment:
Please use lombok `@Getter`

We really need to cut down on boilerplate code. This goes for hash/equals and constructors